.github/workflows/lint.yml

```diff
@@ -9,25 +9,14 @@ on:
       - main
 
 jobs:
-  commitlint:
```

Comment:
let's keep `commitlint`: sadly Trunk doesn't/can't have this functionality, because it has to be executed on a higher level than Trunk operates on, i.e. it should be GitHub-aware, and it's not